added system fonts converted from Ghostscript URW++ L fonts

hastho opened the pull request:
URW++ Design and Development Inc. released their Type 1 implementations of the basic 35 PostScript fonts under the
GNU General Public License and the Aladdin Ghostscript Free Public License.

nschulux commented:
Nimbus Sans L (SANS.FNT) works with Helpfiles.
![Nimbus Sans L](https://github.com/bluewaysw/pcgeos/assets/118665816/19a6c989-ff12-4ee6-bc65-930ac8c70b6c)

nschulux commented:
Can You add Eurosymbol please!
EUROSYMBOL (8364) COPY TO SYMBOL TO 164 WITH FONTFORGE in TrueType Font!

hastho commented:
The font haven't been converted from TrueType into Nimbus-Q. I used the original Type-1 fonts as published by URW++ for Ghostscript and converted them into the proprietary MLS format. Then I used an also proprietary MLS tool to add hints before I converted them into URW Nimbus format, that can be converted using the GeoWorks 1.2 font converter into the final fonts provided here.
So lang answer short: Sorry but I don't want to mess around with additional formats and tools. The intention is just to provide an interim solution before GEOS TrueType support has become production grade.
